sdk/typescript/package.json

```diff
@@ -61,6 +62,11 @@
 			"source": "./src/client/index.ts",
 			"import": "./dist/esm/client/index.js",
 			"require": "./dist/cjs/client/index.js"
+		},
+		"./transactions": {
```

Comment:
I think I like `@mysten/sui.js/transaction` more than`@mysten/sui.js/builder` but happy to use builder instead